.buildkite/pipeline.yml

```diff
@@ -14,6 +14,10 @@ steps:
           rootfs_treehash: "8b5c776db01b54e8632c31fece119b5fe7d86f91"
           # workspaces:
           #   - "/cache/repos:/cache/repos"
+      - JuliaCI/julia#v1:
```

Comment:
Why is this in here twice?

Reply:
The order of operations is:

1. Install Julia in the outer sandbox.
2. Set up the inner sandbox (which requires Julia to set up) and enter the inner sandbox.
3. Install Julia in the inner sandbox.

The inner sandbox is the one that has all the tools (e.g. cmake) from our custom rootfs image. The outer sandbox is very minimal and basically just has the Buildkite agent.

Reply:
I see. It's Julia all the way down.

Reply:
Until you reach turtles.